crypto/message.go

```diff
@@ -88,7 +88,7 @@ func NewPlainMessageFromFile(data []byte, filename string, time uint32) *PlainMe
 // ready for encryption, signature, or verification from an unencrypted string.
 func NewPlainMessageFromString(text string) *PlainMessage {
 	return &PlainMessage{
-		Data:     []byte(strings.ReplaceAll(strings.ReplaceAll(text, "\r\n", "\n"), "\n", "\r\n")),
+		Data:     []byte(internal.CanonicalizeAndTrim(text)),
```

Comment:
we shouldn't trim all inputs. we should only trim when we need a trim. I think only clear-text signatures need to trim.

Reply:
On the BE we convert some instances of signed messages to cleartext according to the preferences, so this is as of now needed for all messages. Please refer to https://github.com/ProtonMail/gopenpgp/pull/136